src/types/contracts/moneyMarket/model.ts

```diff
@@ -0,0 +1,167 @@
+type Pagination = {
+  page: number,
+  page_size: number,
+}
+
+type ContractAndPagination = {
+  address: string,
+  codeHash: string,
+  page?: number,
+  pageSize?: number,
+}
+
+type ParsedPagination<T> = {
+  page: number,
+  pageSize: number,
+  totalPages: number,
+  totalItems: number,
+  data: {
+    [token: string]: T
```

Comment:
this type should be modified or renamed more specifically for the use case, because I know that there are going to be other types of pagination data in the pipeline (swap configs for example) that won't be keyed by a token.